Danki2/Assets/Scripts/AI/IAI.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Assets.Scripts.AI
+{
+    public interface IAI
```

Comment:
I don't think we really need an interface for this? Everything's just using `AI` so can just use that

Reply:
This interface is useful because Enemy can have
```c#
public abstract IAI AI { get; }
```
While target dummy has:
```c#
private AI<TargetDummy> _ai;
public override IAI AI => _ai;
```
It could equally by an abstract class I guess.

Reply:
Edit: I've made it an abstract class instead.